CodeSniffer/Standards/PSR1/Sniffs/Methods/CamelCapsMethodNameSniff.php

```diff
@@ -34,16 +34,6 @@ class PSR1_Sniffs_Methods_CamelCapsMethodNameSniff extends Generic_Sniffs_Naming
 
 
     /**
-     * Constructs a PSR1_Sniffs_Methods_CamelCapsMethodNameSniff.
-     */
-    public function __construct()
```

Comment:
Because it is pointless. The constructor of a parent is not taking any arguments.